Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -486,6 +487,12 @@ public void Detach ()
 			}
 		}
 
+		void StartStepTimer (DebuggerStatistics stats)
+		{
+			stepTimer?.Dispose ();
```

Comment:
If the stepTimer is disposed here then it was not stopped in the normal way and another timer was started before? Is the timing still useful in this case?

Reply:
I don't think this can actually happen since the step buttons become deactivated, but I was covering my bases. If we get any telemetry data with a FailedCount, then we know that we are hitting this.

Reply:
Just to clarify: If the timer gets disposed and the timer was still running, then it will report failure.